### PR TITLE
Add new form for new formats

### LINKED ISSUE #189: [Feature]: form for requesting new format
### What would you like to see added to NeuroConv?

This would be a feature but a specific kind of feature that could use its own issue form, including a request for the link to the project repo and example data. We could then include this form at the end of the gallery so anyone that looks for their format in the gallery and does not find it can submit a request easily.

### Is your feature request related to a problem?

_No response_

### Do you have any interest in helping implement the feature?

No.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
fix #189

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Viewing the true HTML appearance via the GitHub Issues tab is tricky until it's on the `main` branch.

https://github.com/catalystneuro/neuroconv/blob/add_new_format_form/.github/ISSUE_TEMPLATE/format_request.yml is as close of a demo as you can get

File: .github/ISSUE_TEMPLATE/format_request.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    attributes:
+      label: Does the format have any documentation?
+      description: A link or other details to rich documentation describing the folder/file structure.
+  - type: textarea
+    id: example-files
+    attributes:
+      label: Do you have any example files you are willing to share?
+      description: We require small example files for our testing suite to ensure we properly support the format.
+    validations:
+      required: true
```
Comment (bendichter) [inline .github/ISSUE_TEMPLATE/format_request.yml:32]:
```suggestion
```

We do want files but I don't want to make this a requirement of requesting a format interface. We should be able to file a record without an example file on hand.

File: .github/ISSUE_TEMPLATE/format_request.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    attributes:
+      label: What format would you like to see added to NeuroConv?
+      description: What acquisition hardware or software is responsible for producing the output files?
+    validations:
+      required: true
+  - type: textarea
+    id: format-documentation
+    attributes:
+      label: Does the format have any documentation?
+      description: A link or other details to rich documentation describing the folder/file structure.
```
Comment (bendichter) [inline .github/ISSUE_TEMPLATE/format_request.yml:25]:
```suggestion
      description: A link or other details to rich documentation describing the folder/file structure.
    - type: textarea
      id: format-api
      description: Do you know of any open source libraries for reading this data format?
```

Comment (CodyCBakerPhD) [reply .github/ISSUE_TEMPLATE/format_request.yml:25]:
```suggestion
      description: A link or other details to rich documentation describing the folder/file structure.
    - type: textarea
      id: format-api
      attributes:
        label: Existing APIs for format
        description: Do you know of any open source libraries for reading this data format?
```

I'd drop the NEO/SI bit. Without attaching links to those projects they may never have heard of them. Also the backend stuff is something we care about but not necessarily something a simple requester would need to know or check. Once we know the format name and suffixes we can also easily check it ourselves

**Review by bendichter — DISMISSED** [review]

**Pull request merged**